src/sankey.js

```diff
@@ -191,6 +197,39 @@ export default function Sankey(config) {
     }
   }
 
+  /**
+   * Compute nodes with fixed position
+   * @param nodes
+   * @returns {any[]}
+   */
+  function computeFixedNodeLayer({nodes}) {
+    const d3group = group(nodes, d => d.dimIndex);
+    const columnsFlow = [];
+
+    d3group.forEach(el => {
+      columnsFlow.push(el);
+    });
+
+    const x = config.fixedNodeLayerNumber;
+    const kx = (x1 - x0 - dx) / (x - 1);
+
+    const columns = [...columnsFlow];
```

Comment:
When you change `columnsFlow` into `columns`, then these line will be safe to delete I think

Reply:
👍